public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -127,19 +80,24 @@ <h5 class="col-xs-12 box-title">
           </div>
         </div>
         <div class="row">
-          <div class="col-md-4 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
-          <div class="col-md-4 grid-header flex-xs-middle"> Number of particpants </div>
-          <div class="col-md-3 grid-header flex-xs-middle info-text"> <clr-icon class="" shape="info-standard" class="is-solid"></clr-icon> What is SNOMED? </div>
+          <div class="col-md-2 grid-header flex-xs-middle"> {{dbDomain.domainDisplay}}</div>
+          <div class="col-md-4 grid-header flex-xs-middle"> Synonyms </div>
+          <div class="col-md-2 grid-header flex-xs-middle"> Particpants of <b> {{totalParticipants | number}}</b> </div>
+          <div class="col-md-3 grid-header flex-xs-middle info-text">
```

Comment:
Some procedure in concepts seem to have lot of synonyms in there and it looks weird. 
May be we could make vocabulary code 2 ? 
Just a thought. Did not try how that looks for rest of the concepts.
Do not bother if you have tried that out already.

Reply:
Good idea. I did that and made name and synonym take up rest of width. We may add something to hide huge lists of synonyms behind button in future.